Dockerfile

```diff
@@ -1,8 +1,32 @@
-FROM python:3.8.6
+FROM python:3.9.6-alpine3.14
+
+RUN apk update && \
```

Comment:
This may not be incredibly useful since containers are meant to be ephemeral and not managed . . .. just wanted to pass along `apk` virtual groups - https://docs.alpinelinux.org/user-handbook/0.1a/Working/apk.html#_virtuals . It's a nice way to group related packages together to make things easier to update and remove.

Reply:
apk virtual groups could be quite useful, thanks!